integration_tests/.gitignore

```diff
@@ -3,4 +3,5 @@ target/
 dbt_modules/
 logs/
 .DS_Store
-dbt_packages/
\ No newline at end of file
+dbt_packages/
+env/
```

Comment:
I don't have a folder called `env`, but is this meant for when others use it?

Reply:
Yeah! The env is created if you ever create a virtual environment. This will store your entire environment in the folder and is not something we would want to commit to git. Adding it to the .gitignore ensures no virtual environments are committed to the repo.

Reply:
@fivetran-catfritz apologies for not calling this out in the initial comment, can we also add this to the .gitignore for the integration_tests directory.

Reply:
Thanks. Done.